diff --git a/dist/src/vitest-test-runner.js b/dist/src/vitest-test-runner.js
index 84f1ca026503b8e92b7adc952ec6c6a401943656..f9090b6f5e97bdeabd79de69f17d06b562eb2b7d 100644
--- a/dist/src/vitest-test-runner.js
+++ b/dist/src/vitest-test-runner.js
@@ -144,7 +144,7 @@ export class VitestTestRunner {
         if (testIds.length > 0) {
             const parsedTests = testIds.map(fromTestId);
             const regexTestNameFilter = parsedTests
-                .map(({ test: name }) => escapeRegExp(name))
+                .map(({ test: name }) => escapeRegExp(name).replace(/ /g, '(?: > | )'))
                 .join('|');
             const regex = new RegExp(regexTestNameFilter);
             testFilesToRun = parsedTests.map(({ file }) => file);
